### ISSUE TITLE
WCAG 2.3.1 ✅ (Supports): Three Flashes or Below Threshold

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5147]
**Level:** A
**Principle:** Operable
**Result:** ✅  Supports

### Resources
[WCAG 2.2 Success Criterion](https://www.w3.org/WAI/WCAG22/Understanding/three-flashes-or-below-threshold.html)

> [Web pages](https://www.w3.org/WAI/WCAG22/Understanding/three-flashes-or-below-threshold#dfn-web-page) do not contain anything that flashes more than three times in any one second period, or the [flash](https://www.w3.org/WAI/WCAG22/Understanding/three-flashes-or-below-threshold#dfn-flash) is below the [general flash and red flash thresholds](https://www.w3.org/WAI/WCAG22/Understanding/three-flashes-or-below-threshold#dfn-general-flash-and-red-flash-thresholds).

### Audit Checklist
- [x] Review previous Audit Result
- [x] Review work identified in previous audit and list open issues.
- [x] Check for any other relevant open issues.
- [x] Retest open issues, are these issues still present?
- [x] Test for any new issues that may have been introduced.
- [x] Determine compliance status, add this to issue name.
- [x] Add any new issues to the Accessibility project for triage.

### Previous audit
#4130
> ✅ There is no content which flashes. The only moving content is the Carousel which is much slower than three times per second.

 ### Progress since last audit
none.

 ### Audit Notes

There is no content that flashes. 

New functionality has been introduced to to the article page so that when cross-references are used the destination is highlighted briefly to make it easier for users to 'find their place' again.  This is a change of background colour that fades in and out and lasts for two seconds.   It is not expected that this would occur frequently, but rather than users would be reading a few paragraphs, click a cross reference (highlight), read that, click back (highlight) read on. 

Note:
>Since any content that does not meet this success criterion can interfere with a user's ability to use the whole page, all content on the web page (whether it is used to meet other success criteria or not) must meet this success criterion.

So instead of marking this SC as Not-applicable, I will be marking it as Supports.

 ### Work arising

none
